applications/ContactStructuralMechanicsApplication/custom_conditions/mortar_contact_condition.cpp

```diff
@@ -0,0 +1,1014 @@
+// KRATOS  ___|  |       |       |
```

Comment:
This is not detected as a rename for some reason, but it is basically the old ALM_mortar_contact_condition.cpp with some duplicated code removed (that's why is 800 lines lighter)